chatterbot/adapters/logic/closest_match.py

```diff
@@ -4,6 +4,10 @@
 
 
 class ClosestMatchAdapter(BaseMatchAdapter):
+    """
+    The ClosestMatchAdapter extends the BaseMatchAdapter to find
+    the closest matching response.
```

Comment:
This docstring doesn't actually add any information to this class. It just describes what the first line of the class is doing.